### DISCUSSION TITLE
Google Slides embed stopped working

### DISCUSSION THREAD

**Discussion by edu-shamilton1** [jgraph/drawio#4210] [General]
Something seems to have gone wrong.
Currently getting an error when looking for the extension and it has disappeared as an installed plugin in Slides and disappeared from the marketplace.
> 400. That’s an error.
> 
> The server cannot process the request because it is malformed. It should not be retried. That’s all we know.
https://workspace.google.com/marketplace/app/drawio_diagrams_for_slides/588283048931

**Comment by fb-inn** [comment]
Same for Docs, entire draw.io / diagrams.net plugin for Google Docs, Slides seems to have gone.

**Comment by davidjgraph** [comment]
https://github.com/jgraph/drawio/issues/4212
